blueocean-dashboard/src/main/less/forms/index.less

```diff
@@ -5,41 +5,45 @@
     padding-left: 32px;
     padding-top: 26px;
     padding-bottom: 32px;
+    display: flex;
+    justify-content: center;
 
-    h3 {
-        padding-bottom: 10px;
-    }
-
-    .inputControl {
-        display: flex;
-        justify-content: space-between;
-        align-items: center;
-        max-width: 490px;
-    }
-
-    .inputBody .form-element {
-        padding-top: 5px;
-        padding-bottom: 5px;
+    .inputBody {
+        max-width: 400px;
+        min-width: 400px;
```

Comment:
@tfennelly good question. ...but not sure how else to force the inputBody width to be 400px